src/app/system/projects/projects-list/projects-list.component.ts

```diff
@@ -95,12 +95,27 @@ export class ProjectsListComponent implements OnInit {
         return this.session.user.projectAdmin.some(e => e === id);
     }
 
+    /**
+     * Navigate to the project board page
+     *
+     * @param code
+     */
     gotoProjectBoard(code: string) {
         this._router.navigateByUrl('/refresh', { skipLocationChange: true }).then(
             () => this._router.navigate(['/project/' + code])
         );
     }
 
+    /**
+     * Navigate to the collaboration / project members page
+     *
+     * @param code
+     */
+    gotoProjectProjectMembers(code: string) {
```

Comment:
It would be nice if we have a combination with the method "gotoProjectBoard". You can name it for example as openProject(code: string, page: 'collaboration' | 'ontology'). etc. In this case we are more flexibel and can reuse the function. Otherwise we have to add a method for more links. Maybe someone wants a link to the ontology from this list of projects. Do we have to create then an addition method "gotoProjectOntologies"?

There's anyway a typo (project is twice) in the method name:  gotoProjectProjectMembers

Reply:
Do you want to implement now the link to the project ontology page?

Reply:
We can add the link now when it's necessary. It was just an idea to combine methods which do the (almost) same thing. This is why we have methods.

Reply:
methods combination done in 7c9773a